classes/04class/exercises/c04-iac02/gepifanio/variables.tf

```diff
@@ -0,0 +1,47 @@
+variable "region" {
+    type    = string
+    default = "ap-southeast-2"
+}
+
+variable "az_southeast_a" {
+    type    = string
+    default = "ap-southeast-2a"
+}
+
+variable "az_southeast_b" {
+    type    = string
+    default = "ap-southeast-2b"
+}
+
```

Comment:
tip: it is possible to fetch these values and remove them from input:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/availability_zones

Reply:
Awesome!!